src/env.ts

```diff
@@ -55,6 +60,9 @@ const logLevelEnvVar = "DD_LOG_LEVEL";
 const logForwardingEnvVar = "DD_FLUSH_TO_LOG";
 const ddTracingEnabledEnvVar = "DD_TRACE_ENABLED";
 const logInjectionEnvVar = "DD_LOGS_INJECTION";
+const subToApiGatewayLogsEnvVar = "DD_SUBSCRIBE_TO_API_GATEWAY_LOGS";
```

Comment:
I'm not sure DD_SUBSCRIBE_TO_* env vars are useful to the lambda. It doesn't change the behavior of our layers or our extension. Maybe we should remove them?